priam/src/test/java/com/netflix/priam/backup/TestBackup.java

```diff
@@ -156,6 +173,7 @@ private static void generateIncrementalFiles() {
         files.add("target/data/Keyspace1/Standard1/backups/Keyspace1-Standard1-ia-1-Index.db");
         files.add("target/data/Keyspace1/Standard1/backups/Keyspace1-Standard1-ia-2-Data.db");
         files.add("target/data/Keyspace1/Standard1/backups/Keyspace1-Standard1-ia-3-Data.db");
+        // purposely testing case mismatch in secondary index directory (CASS-2201)
```

Comment:
nit: good to avoid internal ticket references in oss

Reply:
Done